mod_ci/controllers.py

```diff
@@ -75,6 +93,11 @@ def kvm_processor(db, kvm_name, platform, repository, delay):
         log.debug('[{platform}] Sleeping for {time} seconds'.format(
             platform=platform, time=delay))
         time.sleep(delay)
+    maintenance_mode = MaintenanceMode.query.filter(MaintenanceMode.platform ==
+                                                    platform).first()
+    if maintenance_mode is None or maintenance_mode.mode == 'True':
```

Comment:
So as long as I never switch maintenance mode on/off for the first time, a platform will never run tests?